src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -161,7 +160,7 @@ void StartBackgroundThread()
 		{
 			backgroundThread = new HandlerThread("CameraBackground");
 			backgroundThread.Start();
-			backgroundHandler = new Handler(backgroundThread.Looper);
+			backgroundHandler = new Handler(backgroundThread.Looper ?? throw new NullReferenceException());
```

Comment:
It's not. 

This just let's the Roslyn analyzer know that we are expecting a NullReferenceException here.

The Xamarin.Android team made `Looper` nullable, and so we don't really have any other option here because we need to use `Looper`. 

After Xamarin.Android is initialized, `Looper` should never be null, so this code should never throw a `NullReferenceException`.